proposals/description/eth_lbp.ts

```diff
@@ -0,0 +1,64 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const eth_lbp: ProposalDescription = {
+  title: 'FIP-110: Reinforce PCV via ETH LBP and tighter spread',
+  commands: [
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'setSafeAddresses(address[])',
+      arguments: [['{ethToDaiLBPSwapper}']],
+      description: 'Set the ethToDai LBP swapper to be guardian Safe addresses'
+    },
+    ////////    ETH LBP    ////////
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: ['{aaveEthPCVDeposit}', '{ethToDaiLBPSwapper}', '20000000000000000000000', false, false],
+      description: 'Transfer WETH from Aave to the LBP swapper'
+    },
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: ['{compoundDaiPCVDeposit}', '{ethToDaiLBPSwapper}', '3000000000000000000000000', false, false],
+      description: 'Withdraw 3M DAI from the CompoundPCVDeposit and transfer to the LBP pool'
+    },
+    // Swap and update CR oracle
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'swap()',
```

Comment:
Makes sense to use `swap()`. Duration is 2 days, and TC timelock takes 4 days. `forceSwap()` is `onlyGovernor`